### ISSUE TITLE
(suggestion) High voltage substation switches that arc!

### ISSUE THREAD

**Issue by pooing1234** [Electrical-Age/ElectricalAge#646]
I think it would be cool to have something like this https://www.youtube.com/watch?v=MAlyEMQxTN0

A switch like this can be used for extremely high voltages in a grid, but can cause huge arcs while under load.

**Comment by Baughn** [comment]
Yup, I've been thinking the same thing. I guess the model shouldn't be too difficult to make, though the arcing will be fun.

**Comment by cm0x4D** [comment]
The idea is not completely new, in #591 we did already discuss such a high voltage switch. 

IRL those are controlled by different protection devices (called IED) offering different functions like protecting the lines from to high currents like breakers do on normal installations or protect a part of the grid from over-voltage or protect a transformer from being overpowered and so on...

I propose to have a generic high voltage switch which's functionality can be determined by either placing different items (like the fuel heat furnace has multiple burners that can be installed) to the switche's inventory or even a simpler Solution would be that the switch has an Signal voltage input that actually controls the switch. The second would be probably simpler to implement and maybe more fun to play with...
